uint16[] drive_ids
string[] mode
---
bool[] drives_success